medical-clinic/src/main/java/com/Mafiuz04/medicalclinic/service/PatientService.java

```diff
@@ -19,27 +20,38 @@ public class PatientService {
     private final PatientMapper patientMapper;
     private final JPAPatientRepository patientRepository;
 
+    //TC1: W przypadku istnienia listy pacjentów, zostaje ona zwrócona.
     public List<PatientDto> getPatients(Pageable pageable) {
         List<Patient> patients = patientRepository.findAll(pageable).getContent();
         return patientMapper.listToDto(patients);
     }
 
+    //TC1: W przypadku gdy pacjent o danym ID istnieje, zostanie przemapowany i zwrcony.
+    //TC2: W przypadku gdy pacjent o danym ID nie istnieje, zostanie rzucony wyjątek.
     public PatientDto getPatientById(Long id) {
         return patientMapper.toDto(patientRepository.findById(id)
                 .orElseThrow(() -> new MedicalClinicException("There is no patient with given email.", HttpStatus.BAD_REQUEST)));
     }
 
-    public PatientDto addPatient(Patient patient) {
-        checkData(patient);
-        ifGivenEmailExist(patient);
-        ifGivenIdNumberExist(patient);
+    public PatientDto addPatient(PatientCreateDto patientCreateDto) {
+//        checkData(patientCreateDto);
+//        ifGivenEmailExist(patientCreateDto);
+//        ifGivenIdNumberExist(patientCreateDto);
+        Patient patient = patientMapper.createToPatient(patientCreateDto);
         return patientMapper.toDto(patientRepository.save(patient));
     }
 
+    //TC1: W przypadku istnienia pacjenata o danym ID, zostanie on usnięty z reposytorium.
     public void deletePatient(Long id) {
         patientRepository.deleteById(id);
     }
 
+    //TC1: W przypadku gdy pacient o danym ID istnieje, wszystkie dane są uzupełnione, numer dowodu nie został zmieniony
```

Comment:
ja bym to troche bardziej rozszerzyl bo zauwaz ze uzywasz metody   isItExistingPatient a nie wypisales jej w tc1